TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+**AmpliPi™** is a trademark of **MicroNova LLC**. You may not use this trademark without prior express written permission of MicroNova LLC and pursuant to the trademark policy shown below. This unmodified file must be included with all copies of the AmpliPi software.
+
+Version 1.0  
+Last Modified: March 8, 2021
+
+# Table of Contents
+
+- [Introduction](#Introduction)
+- [Our Commitment to Open Source Principles](#Our-Commitment-to-Open-Source-Principles)
+- [Trademarks Subject to the Guidelines](#Trademarks-Subject-to-the-Guidelines)
+- [Universal Considerations for All Uses](#Universal-Considerations-for-All-Uses)
+- [Use for Software](#Use-for-Software)
+- [Use for Non-Software Goods and Services](#Use-for-Non-Software-Goods-an-Services)
+- [General Considerations About Trademarks and Their Use](#General-Considerations-About-Trademarks-and-Their-Use)
+- [Where to Get Further Information](#Where-to-Get-Further-Information)
+
+# Introduction
+
+This document, the "Policy," outlines the AmpliPi project's (the "Project") policy for the use of our trademarks. While our software is available under a free and open source software license, the copyright license does not include an implied right or license to use our trademark.
+
+The role of trademarks is to provide assurance about the quality of the products or services with which the trademark is associated. But because an open source license allows your unrestricted modification of the copyrighted software, we cannot be sure that your modifications to the software are ones that will not be misleading if distributed under the same name. Instead, this Policy describes the circumstances under which you may use our trademarks.
+
+In this Policy we are not trying to limit the lawful use of our trademarks, but rather describe for you what we consider the parameters of lawful use to be. Trademark law can be ambiguous, so we hope to provide enough clarity for you to understand whether we will consider your use licensed or non-infringing.
+
+The sections that follow describe what trademarks are covered by this Policy, as well as uses of the trademarks that are allowed without additional permission from us. If you want to use our trademarks in ways that are not described in this Policy, or if you are uncertain about any of the following information contained in this Policy, please see "Where to get further information" below for contact information. Any use that does not comply with this Policy or for which we have not separately provided written permission is not a use that we have approved.
```

Comment:
"Where to get further information" should be formatted like it is below, IMO. "Where to Get Further Information"